### DISCUSSION TITLE
Have it fixed the bug in pin to start on Win11?

### DISCUSSION THREAD

**Discussion by YeFeng2003** [valinet/ExplorerPatcher#2004] [Q&A]
Such as problem

**Comment by pyrates999** [comment]
work around listed here: https://github.com/valinet/ExplorerPatcher/discussions/1679

**Comment by milestprower92** [comment]
The Only Fix For the Pin To Start Button is Using The "UndockingDisabled" Registry Tweak But Search Breaks
